### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Connect to Local Server_Keyboard: Focus is not visible on checkbox, "Connect" button and "Remote Server or manual setup" button.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#96]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated:** MAS2.4.7

**Repro Steps:**
1. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2. Download and install "Microsoft IIS Administration" application.
3. Tab till "Skip This" button and Press Enter.
4. Verify that keyboard focus is visible clearly on each interactive control.

**Actual Result:**
Focus is not visible on checkbox, "Connect" button and "Remote Server or manual setup" button

**Expected Result:**
When any interactive control receives keyboard focus, a rectangular border should appear around these controls so that user can differentiate clearly between the focused and non-focused controls on the page. 

**User Impact:**
This Success Criterion helps anyone who relies on the keyboard to operate the page, by letting them visually determine the component on which keyboard operations will interact at any point in time.
People with attention limitations, short term memory limitations, or limitations in executive processes benefit by being able to discover where the focus is located.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Attachment:**
![bug 96_focus visibility issue](https://user-images.githubusercontent.com/38521133/44655752-e05b9900-aa13-11e8-8b61-97e1c6144a39.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Ashwinipola26** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #C+E-H; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #MAS2.4.7; #Keyboard; #.NETWebsites; #Edge; #Chrome

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev1;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
